include/RAJA/policy/sycl/kernel/SyclKernel.hpp

```diff
@@ -126,25 +108,18 @@ void SyclKernelLauncher(Data data, cl::sycl::nd_item<3> item)
  * Helper class that handles SYCL kernel launching, and computing
  * maximum number of threads/blocks
  */
-template<typename LaunchPolicy, typename StmtList, typename Data, typename Types>
+template<bool, typename LaunchPolicy, typename StmtList, typename Data, typename Types>
```

Comment:
@homerdin, I'd prefer to have a name here for the bool so we have a trail later, would you mind a change like this?

```suggestion
template<bool IsTriviallyCopyable, typename LaunchPolicy, typename StmtList, typename Data, typename Types>
```

Reply:
Good suggestion. 👍